src/figuresClasses.ts

```diff
@@ -1,11 +1,81 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: () => number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [this.a, this.b, this.c].sort((x, y) => y - x);
```

Comment:
You may skip it - but I would avoid `sort` here

You may look at `Math.max` instead

Reply:
Can show me how to do that with Math.max, please)

Reply:
I want to see your solution with Math.max because in my a lot of lines

Reply:
I chose sort because this way I have access not only to the largest side but also to all the others